backend/daily_summary/repositories/composed_meal_items_repository.py

```diff
@@ -17,6 +17,11 @@ def __init__(self, db: AsyncSession):
     async def get_composed_meal_item_by_id(self, composed_meal_item_id: UUID) -> ComposedMealItem | None:
         return await self.db.get(ComposedMealItem, composed_meal_item_id)
 
+    async def get_composed_meal_item_by_meal_id(self, meal_id: UUID) -> ComposedMealItem | None:
+        query = select(ComposedMealItem).where(ComposedMealItem.meal_id == meal_id)
+        result = await self.db.execute(query)
+        return result.scalar_one_or_none()
```

Comment:
Remove this method or refactor to sth like that:
```suggestion
    async def get_composed_meal_items_by_meal_id(self, meal_id: UUID) -> List[ComposedMealItem] | None:
        query = select(ComposedMealItem).where(ComposedMealItem.meal_id == meal_id)
        result = await self.db.execute(query)
        return result.scalars().all()
```

Reply:
I thought I removed it. It is not used anywhere so will be removed.

Reply:
It is still unchanged